app/models/organization_content.rb

```diff
@@ -5,4 +5,5 @@ class OrganizationContent < ActiveRecord::Base
   validates_length_of :title, maximum: 255
   validates_length_of :description, maximum: 300
   validates_length_of :introduction, maximum: 5000
+  validates_length_of :announcement, maximum: 5000
```

Comment:
5K chars seems like it would take up a chunk of UI with text. maybe 2k limit?

Reply:
5000 is what we gave both introduction and description. I feel like a character limit should be enforced on the front end before it hits a max and the request fails opaquely.